lib/gitter-adapter.js

```diff
@@ -216,10 +219,25 @@ Adapter.prototype.subscribeToRoom = function(room) {
 };
 
 Adapter.prototype.sendMessage = function(target, message) {
+  this.sendQueue.push({"target": target, "message": message});
+  this.checkSendQueue();
+};
+
+Adapter.prototype.checkSendQueue = function() {
+  if (!this.sending && this.sendQueue.length >= 1) {
+    var message = this.sendQueue.shift();
+    this.actualSendMessage(message["target"], message["message"]);
+  }
+};
+
+Adapter.prototype.actualSendMessage = function(target, message) {
```

Comment:
Let's put this back to `sendMessage` and rename the queueing function to `queueMessage`